### PR TITLE
Add missing methods for FormData type

### LINKED ISSUE #14813: Property 'get' does not exist on type 'FormData'.
Expected behavior:**
[(https://developer.mozilla.org/en-US/docs/Web/API/FormData/get)](https://developer.mozilla.org/en-US/docs/Web/API/FormData)
The above method tells about methods which can be used on formData.
**Actual behavior:**
Throws error 

> Property 'get' does not exist on type 'FormData'.)

### PR DESCRIPTION
**Pull request by NColey**
Fixes #14813.

### CODE REVIEW

**Comment by mhegazy** [comment]
Thanks for your PR. these files are not manually edited, they are auto-generatd from a script in . You can find more information about contributing lib.d.ts fixes at https://github.com/Microsoft/TSJS-lib-generator. 

Changes should be done in https://github.com/Microsoft/TSJS-lib-generator instead.

Please find more information at: https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes.

**Comment by mhegazy** [comment]
Your changes to src/lib/DOM.iterable.d.ts we should take, please revert the other changes and send them to TSJS-lib-generator instead.

**Comment by NColey** [comment]
Hi @mhegazy I reverted the unnecessary changes leaving just the updates to src/lib/DOM.iterable.d.ts. Let me know if there is anything else I should do, thanks!

File: src/lib/dom.iterable.d.ts [hunk 1]
```diff
@@ -4,6 +4,23 @@ interface DOMTokenList {
     [Symbol.iterator](): IterableIterator<string>;
 }
 
+interface FormData {
+    /**
+     * Returns an array of key, value pairs for every entry in the list
+     */
+    entries(): IterableIterator<[string, any]>;
```
Comment (mhegazy) [inline src/lib/dom.iterable.d.ts]: from [spec](https://xhr.spec.whatwg.org/#formdataentryvalue), the values are `string | File`; that applies to `value`, `entries` and `[Symbol.iterator]`.

Comment (NColey) [inline src/lib/dom.iterable.d.ts]: I just updated the PR with the appropriate types, thanks!

**Comment by mhegazy** [comment]
thanks!

**Pull request merged**